.gitmodules

```diff
@@ -34,3 +34,6 @@
 [submodule "lib/graphql/GraphQL-interface"]
   path = lib/graphql/GraphQL-interface
   url = https://github.com/CanDIG/GraphQL-interface.git
+[submodule "lib/vault/Vault-Helper-Tool"]
```

Comment:
Installing via `go install` is sufficient. In this case I would suggest not including the helper tool as a submodule.